db/seeds.rb

```diff
@@ -8,12 +8,71 @@
 
 require 'open-uri'
 
-puts 'Creating volunteers'
+
+# --------------- COUNTRIES ---------------
+
+puts "Cleaning COUNTRIES database..."
+Country.destroy_all
+
+puts "Creating countries..."
+COUNTRIES = ['Europe', 'Austria', 'Belgium', 'Bulgaria', 'Croatia', 'Republic of Cyprus', 'Czech Republic', 'Denmark', 'Estonia', 'Finland', 'France', 'Germany', 'Greece', 'Hungary', 'Ireland', 'Italy', 'Latvia', 'Lithuania', 'Luxembourg', 'Malta', 'Netherlands', 'Poland', 'Portugal', 'Romania', 'Slovakia', 'Slovenia', 'Spain', 'Sweden']
+COUNTRIES.each do |country|
+  Country.new(name: country).save # Add flags ?
```

Comment:
I think we should add flags yes, question is, how? could we add it as an emoji? 😅 on a string? that would be nice. Else, as a picture I guess. If we add as a picture, better not to use all the countries for the seeds (after all, in reality we will only have Europe, Germany, Portugal and Italy to start with), as adding photos really increases the time it takes to seed